modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -188,6 +188,15 @@ enum VideoCaptureProperties {
        CAP_PROP_HW_ACCELERATION_USE_OPENCL=52, //!< (**open-only**) If non-zero, create new OpenCL context and bind it to current thread. The OpenCL context created with Video Acceleration context attached it (if not attached yet) for optimized GPU data copy between HW accelerated decoder and cv::UMat.
        CAP_PROP_OPEN_TIMEOUT_MSEC=53, //!< (**open-only**) timeout in milliseconds for opening a video capture (applicable for FFmpeg back-end only)
        CAP_PROP_READ_TIMEOUT_MSEC=54, //!< (**open-only**) timeout in milliseconds for reading from a video capture (applicable for FFmpeg back-end only)
+       CAP_PROP_VIDEO_STREAM = 55, //!< Default value is 0 (based index), -1 is disable. If used for devices, then -1 - camera is off, 0 - camera is on
+       CAP_PROP_AUDIO_STREAM = 56, //!< Specify stream in multi-language media files, -1 - disable audio processing (default). If used for devices, then -1 - microphone is off, 0 - microphone is on
+       CAP_PROP_AUDIO_POS = 57, //!< Audio position is measured in samples. Accurate audio sample timestamp of previous grabbed fragment. See CAP_PROP_AUDIO_SAMPLES_PER_SECOND
+       CAP_PROP_AUDIO_DATA_DEPTH = 58, //!< Alternative definition to bits-per-sample, but with clear handling of 32F / 32S
+       CAP_PROP_AUDIO_SAMPLES_PER_SECOND = 59, //!< determined from file/codec input. If not specified, then selected audio sample rate is 44100
+       CAP_PROP_AUDIO_BASE_INDEX = 60, //!< Number of video channels
```

Comment:
> Number of video channels

(read-only) Index of first audio channel. Used as parameter for `.retrieve()` call.


---

Perhaps we should have: `CAP_PROP_VIDEO_TOTAL_CHANNELS` (currently with the same value)

Reply:
Agree, it's not clear why Audio_base_index sets number of *video* channels

Reply:
What do I have to do here: to rename CAP_PROP_AUDIO_BASE_INDEX=>CAP_PROP_VIDEO_TOTAL_CHANNELS or to add a new CAP_PROP_VIDEO_TOTAL_CHANNELS property and to change the CAP_PROP_AUDIO_BASE_INDEX description?

Reply:
I would do both. (add total and rename to VIDEO_TOTAL_CHANNELS). Also, you need to define clearly in docs that audio channel number is not zero-indexed, it continues enumeration after video channels.

Reply:
> > Number of video channels
> 
> (read-only) Index of first audio channel. Used as parameter for `.retrieve()` call.
> 
> Perhaps we should have: `CAP_PROP_VIDEO_TOTAL_CHANNELS` (currently with the same value)

What do I have to do here: to rename CAP_PROP_AUDIO_BASE_INDEX=>CAP_PROP_VIDEO_TOTAL_CHANNELS or to add a new CAP_PROP_VIDEO_TOTAL_CHANNELS property and to change the CAP_PROP_AUDIO_BASE_INDEX description?

Reply:
- `CAP_PROP_AUDIO_BASE_INDEX` should be here with updated comment (see above).
- add new property `CAP_PROP_VIDEO_TOTAL_CHANNELS` (for RGBD or stereo streams/cameras)